### PR TITLE
Show success flash after editing appointment notes

### PR DESCRIPTION
**Pull request by benilovj**
Display a banner to confirm that the user has successfully updated the appointment notes:

image

## Changes

- Update `govuk-frontend` to v3.11 in order to pick up the [notifications banner component](https://design-system.service.gov.uk/components/notification-banner/)
- Add the `connect-flash` package, which introduces flash middleware into the app
- Wire in the success banner on the appointment page

## Questions

Do we need this functionality at all?
Is the banner displayed in the right place on the page?

### CODE REVIEW

File: app/views/case/appointment.html [hunk 1]
```diff
@@ -6,6 +6,11 @@
 
 {% block content %}
 
+{{ govukNotificationBanner({
+  text: messages[0],
+  type: 'success'
+}) if messages.length > 0 }}
+
```
Comment (fofr) [inline app/views/case/appointment.html:13]: Should this be in a layout file?

**Review by fofr — APPROVED** [review]
1 question, but doesn't need to block

File: app/views/case/appointment.html (hunk 1 above)
Comment (benilovj) [reply app/views/case/appointment.html:13]: Yeah, I wasn't sure about the positioning of it. Where would you recommend to put it?

Comment (benilovj) [reply app/views/case/appointment.html:13]: We spoke offline, and decided to fix this on a separate PR after the first MO.

**Pull request merged**